sensor/src/types/config.py

```diff
@@ -1,59 +1,69 @@
-from typing import Literal
-import attrs
-import attrs.validators as val
+from typing import Any, Literal
+import attr
```

Comment:
Your old variant `attrs` [is preferred](https://www.attrs.org/en/latest/names.html#tl-dr) over `attr`.